test/testdata/decorder.go

```diff
@@ -0,0 +1,19 @@
+// args: -Edecorder
+//config_path: testdata/configs/decorder.yml
+package testdata
+
+const (
+	decoc = 1
```

Comment:
```suggestion
	decoc = math.MaxInt
```

add the import:
```
import "math"
```

We add one import of stdlib by test because some side effects can be hidden without that.

Reply:
Suggestion is applied